# Gross Income And Exclusions

**Selected Excercises**

McGraw Hill's Taxation of Individuals and Business Entities 2025

Spilker, Ayers, Lewis, Weaver, Barrick, Robinson, Worsham

In [ ]:
import os
import sys
import pandas as pd
from datetime import date
from IPython.display import display, HTML

mod_path = os.path.abspath(os.path.join(".."))
if mod_path not in sys.path:
    sys.path.append(mod_path)


---

##### **For the following independent cases, determine whether economic income is present and, if so, whether it must be included in gross income (i.e., is it realized and recognized for tax purposes?)**

- **a) Asia owns stock that is listed on the New York Stock Exchange, and this year the stock increased in value by 20,000.**

**Answer:**

- Economic income: Yes - 20,000 increase in wealth.
- Realized: No - no sale, exchange, or other triggering event.
- Recognized: No - because it wasn't realized

- **b) Ben sold stock for 10,000 and paid a sales commission of 250. Ben purchased the stock several years ago for 4,000.**

In [24]:
realized = 10000
cost = 4000
commission = 250
recognized = realized - cost - commission

print(f"""There is Economic Benefit:
      
    Amount realized ({realized:,.0f} realized - {commission:,.0f} commission): {realized - commission:,.0f}
    Basis:                                             ({cost:,.0f})
                                                       -------
    Realized & Recognized LT Capital Gain:              {recognized:,.0f}""")

There is Economic Benefit:

    Amount realized (10,000 realized - 250 commission): 9,750
    Basis:                                             (4,000)
                                                       -------
    Realized & Recognized LT Capital Gain:              5,750


- **c) Bessie is a partner in SULU Enterprises LLC. This year SULU reported that Bessie’s share of rental income was 2,700 and her share of municipal interest was 750.**

In [26]:
rental_income = 2700
municipal_interest = 750

print(f"""There is Economic Benefit:
    Realized:
        Rental income        {rental_income:,.0f}
        Municipal interest     {municipal_interest:,.0f}
    Total Realized           {rental_income + municipal_interest:,.0f}

    Recognized:
        Rental Income        {rental_income:,.0f}

Both are examples of economic benefits. Because municipal interest 
is explicitly excluded by tax law from gross income it is not recognized.
""")

There is Economic Benefit:
    Realized:
        Rental income        2,700
        Municipal interest     750
    Total Realized           3,450

    Recognized:
        Rental Income        2,700

Both are examples of economic benefits. Because municipal interest 
is explicitly excluded by tax law from gross income it is not recognized.



---

##### **Devon owns 1,000 shares of stock worth 10,000. This year he received 200 additional shares of this stock from a stock dividend. His 1,200 shares are now worth 12,500. Must Devon include the dividend paid in stock in income?**

**Answer:**

**IRC 305**(a) provides the basic rule:

"Gross income does not include the amount of any distribution of the stock of a corporation made by such corporation to its shareholders with respect to its stock".

The stock dividend does not represent a taxable event, hence Devon does not include it in gross income.


---

##### **XYZ declared a 1 USD per share dividend on August 15. The date of record for the dividend was September 1 (the stock began selling ex-dividend on September 2). The dividend was paid on September 10. Ellis is a cash-method taxpayer. Determine if he must include the dividends in gross income under the following independent circumstances:**

- **a) Ellis bought 100 shares of XYZ stock on August 1 for 21 per share. Ellis received a 100 dividend on September 10. Ellis still owns the shares at year-end.**

    - **Answer:** Yes. Ellis must include the 100 dividend in gross income.

- **b) Ellis bought 100 shares of XYZ stock on August 1 for 21 per share. Ellis sold his XYZ shares on September 5 for 23 per share. Ellis received the 100 dividend on September 10 (note that even though Ellis didn’t own the stock on September 10, he still received the dividend because he was the shareholder on the record date).**
    
    - **Answer:** Yes. Ellis must include the 100 dividend in gross income.

- **c) Ellis bought 100 shares of XYZ stock for 22 per share on August 20. Ellis received the 100 dividend on September 10. Ellis still owns the shares at year-end.**

    - **Answer:** Yes. Ellis must include the 100 dividend in gross income.

---

##### **For the following independent cases, determine whether economic income is present and, if so, whether it must be included in gross income. Identify a tax authority that supports your analysis.**

- **Answer:** The following will apply to the next set of questions:
    - **IRC 61:** *"Except as otherwise provided in this subtitle, gross income means all income from whatever source derived..."*

- **a) Hermione discovered a gold nugget (valued at 10,000) on her land.**

    - **Answer:** Yes. Hermione must include the value of the gold nugget found (10,000). This falls under the "treasure trove" doctrine, specifically Reg.1.61-14, which explicitly states treasure trove is includible in gross income for the year it's reduced to undisputed possession, at its fair market value.

- **b) Jay embezzled 20,000 from his employer and has not yet been apprehended.**

    - **Answer:** Yes. Jay must include the 20,000 stolen from his employer.

- **c) Keisha found 1,000 inside an old dresser. She purchased the dresser at a discount furniture store at the end of last year and found the money after the beginning of the new year. No one has claimed the money.**

    - **Answer:** Yes. Keisha must include the 1,000 in the year it was found. Reg 1.61-14 also apply here.

---

##### **Although Hank is retired, he is an excellent handyman and often works part time on small projects for neighbors and friends. Last week his neighbor, Mike, offered to pay Hank 500 for minor repairs to his house. Hank completed the repairs in December of this year. Hank uses the cash method of accounting and is a calendar-year taxpayer. Compute Hank’s gross income for this year from each of the following alternative transactions:**

- **a) Mike paid Hank 200 in cash in December of this year and promised to pay the remaining 300 with interest in three months.**
    - **Answer:** Hank's gross income would be the 200 received in December.  Then, on the next tax year he would report the remaining 300 and whatever interest generated as interest income.

- **b) Mike gave Hank tickets in December to the big game in January. The tickets have a face value of 50, but Hank could sell them for 400. Hank went to the game with his son.**
    - **Answer:** Hank's gross income would include 400, the market value of the ticket.

- **c) Mike bought Hank a new set of snow tires and gave them to Hank in December. The tires typically sell for 500, but Mike bought them on sale for 450.**
    - **Answer:** Hank should report the market value of the snow tires as gross income: 500.

---

##### **Jim recently joined the Austin Barter Club, an organization that facilitates the exchange of services between its members. This year Jim provided lawn-mowing services to other club members. Jim received the following from the barter club. Determine the amount, if any, Jim should include in his gross income in each of the following situations:**

- **a) Jim received 275 of car repair services from another member of the club.**
    - **Answer:** Jim should include 275 as gross income.

- **b) Jim received a 550 credit that gave him the option of receiving a season pass at a local ski resort from another member of the club. However, he forgot to request the pass by the end of the ski season and his credit expired.**
    - **Answer:** Jim should include 550 as gross income.

- **c) Jim received a 450 credit that can only be applied for goods or services from club members next year.**
    - **Answer:** Jim should include 450 in gross income this year.

---

##### **Last year Acme paid Ralph 15,000 to install a new air-conditioning unit at its headquarters building. The air conditioner did not function properly, and this year Acme requested that Ralph return the payment. Because Ralph could not repair one critical part in the unit, he refunded the cost of the repair, 5,000, to Acme.**

- **a) Is Ralph required to include the 15,000 payment he received last year in his gross income from last year?**
    - **Answer:** Yes, Ralph is required to include the 15,000 payment in gross income.

- **b) What are the tax implications of the repayment if Ralph was in the 35 percent tax bracket when he received the 15,000 payment from Acme but he was in the 24 percent tax bracket when he refunded 5,000 to Acme?**

In [29]:
last_year_tax_rate = .35
this_year_tax_rate = .24
this_year_refund = 5000

# Option A: Deduction in the repayment year
option_a = this_year_refund * this_year_tax_rate

# Option B: Take a credit under IRC 1341
option_b = this_year_refund * last_year_tax_rate

relief = max(option_a, option_b)

print(f"""
Under IRC 1341, when a taxpayer included an amount
in gross income in an earlier year because it appeared
to be his right to the money, but later discovers that 
he did not have an unrestricted right to that amount and
repays it, he may receive relief.

Here Ralph has two options:

      Option A: Deduction in the repayment year:   $ {option_a:,.0f}
      Option B: Take a credit under IRC 1341:      $ {option_b:,.0f}

Since Option B offers the best outcome, he chooses Option B.""")


Under IRC 1341, when a taxpayer included an amount
in gross income in an earlier year because it appeared
to be his right to the money, but later discovers that 
he did not have an unrestricted right to that amount and
repays it, he may receive relief.

Here Ralph has two options:

      Option A: Deduction in the repayment year:   $ 1,200
      Option B: Take a credit under IRC 1341:      $ 1,750

Since Option B offers the best outcome, he chooses Option B.
